src/Components/TodoList.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import Todo from "./Todo";
+
+// Todo List
+const TodoList = ({ todoList, remove, edit }) => {
```

Comment:
Prop types define the expected data type of props. It shows error if some other type of value is passed in dev mode. Default props are used to assign default values to props in case they are not passes from the parent component.